src/etl/adapter/repository.py

```
@@ -60,32 +71,46 @@ async def save_batch(self, messages: List[MessageMetadata]) -> None:
-}
+                    }
                 )
                 for vector, msg in zip(embeddings, messages)
             ]
+
             self._client.upload_points(
                 collection_name=self._collection_name,
                 points=points,
                 wait=True
             )
         except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)